test/unit/init/AclInitializer.test.ts

```diff
@@ -61,21 +60,19 @@ describe('AclInitializer', (): void => {
   });
 
   it('does not invoke ACL initialization when a root ACL already exists.', async(): Promise<void> => {
-    store.getRepresentation.mockReturnValueOnce(Promise.resolve({
-      data: { destroy: jest.fn() },
-    } as any));
+    (store.resourceExists as jest.Mock).mockImplementationOnce((): any => true);
```

Comment:
Easier: `.mockResolvedValueOnce(true)`. Also the cast should not be necessary as the store is typed as a mock above.